pkgs/top-level/aliases.nix

```diff
@@ -211,6 +211,7 @@ mapAliases ({
   hicolor_icon_theme = hicolor-icon-theme; # added 2018-02-25
   htmlTidy = html-tidy;  # added 2014-12-06
   iana_etc = iana-etc;  # added 2017-03-08
+  iasl = acpica-tools;  # added 2020-10-27
```

Comment:
I wouldn't mark this as an alias but throw an error that it was removed.

Reply:
Is there any documentation / guidance on when to use either of the two options (alias vs. error)? Intuitively I went with the alias solution since it's a drop-in replacement (acpica-tools is a superset of iasl) and we don't know how many references there are in users' configurations.

Reply:
Generally an error is thrown when the package is removed to notify the user of the removal. Otherwise their configs could throw errors which can be hard to track down. I would write in the message that ``acpica-tools`` is a drop in replacement and should work without any changes. The problem is that we don't know if it works in every use case and people could be surprised by the sudden change they where not informed about.

Reply:
In this particular case I'd be hard stretched to think of a reason why the replacement wouldn't work.

iasl and acpica-tools are literally the same thing, we used to have two derivations built from the same source, it's just that the iasl derivation only kept one of the multiple built binaries, whereas acpica-tools kept all of them except iasl. The ${pkg}/bin/iasl is exactly the same for pkg = iasl and pkg = acpica-tools.

I won't push back any further if you don't think this explanation is convincing, I don't particularly feel strongly either way.

Reply:
There are reasons to avoid aliases altogether: Fixing a config with a clearly thrown error has very little cost, while multiple names for the same thing just add unnecessary indirection / confusion. I don't feel that strongly about it either though, and there is plenty of precedent for both options.